src/material-experimental/mdc-select/select.html

```diff
@@ -1,14 +1,18 @@
 <!--
- Note that the select trigger element specifies `aria-owns` pointing to the listbox overlay.
+ Note 1: that the select trigger element specifies `aria-owns` pointing to the listbox overlay.
  While aria-owns is not required for the ARIA 1.2 `role="combobox"` interaction pattern,
  it fixes an issue with VoiceOver when the select appears inside of an `aria-model="true"`
  element (e.g. a dialog). Without this `aria-owns`, the `aria-modal` on a dialog prevents
  VoiceOver from "seeing" the select's listbox overlay for aria-activedescendant.
  Using `aria-owns` re-parents the select overlay so that it works again.
  See https://github.com/angular/components/issues/20694
+
+ Note 2: we set `role="presentation"`, because the trigger has some CSS that sets it to
+ `display: inline-table` which causes it to be read out as a table by NVDA on Chrome (see #21480).
 -->
 <div cdk-overlay-origin
      [attr.aria-owns]="panelOpen ? id + '-panel' : null"
+     role="presentation"
```

Comment:
@crisbeto It seems axe is not happy that this has both `role=presentation` _and_ `aria-owns`

Reply:
Do we have the option of excluding it? I don't get the error when running the axe extension locally so it's either an old version or an option that people have to opt into. The alternative would be to remove the `display: inline-table` from the CSS, but I suspect that'll cause a bunch of screenshot diff failures instead.

Reply:
This is the rule that's triggered: https://dequeuniversity.com/rules/axe/4.1/presentation-role-conflict. It's listed as a minor thing, so I'm guessing maybe the app has stricter rules enabled than we do